docs/DeveloperGuide.adoc

```diff
@@ -844,16 +901,67 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+[discrete]
+=== Use case: Sort by locality
+
+*MSS*
+
+1.  User types “/sortl” in to the command line.
+2.  Addressbook will prompt user to select whether they would like to view the list for students or tutors.
```

Comment:
No longer `Addressbook`, remember to change all other use cases too

Reply:
Will update in my commit